index.js

```diff
@@ -22,4 +22,38 @@ addTaskButton.addEventListener("click", () => {
   });
 
   console.log(tasks);
+
+  taskTable = document.getElementById("task-table");
+  const TaskDetails = document.createElement("tr");
+  TaskDetails.className = "table-box-row";
+
+  for (let taskArrayIndex in tasks) {
+    for (
+      let taskObjectIndex = 0;
+      taskObjectIndex < Object.values(tasks[taskArrayIndex]).length;
+      taskObjectIndex++
+    ) {
```

Comment:
Is this really necessary? Couldn't you just loop through array of objects by a simple `forEach` ?

Reply:
Great, I searched for that and got It's much better to use "forEach" instead of "for" for 2 reasons:
1) It's more optimal.
2)It's better when you need all values in array